pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1675,9 +1675,9 @@ func (l *LibvirtDomainManager) GetGuestInfo() (v1.VirtualMachineInstanceGuestAge
 }
 
 // InterfacesStatus returns the interfaces Guest Agent reported
-func (l *LibvirtDomainManager) InterfacesStatus(domainInterfaces []api.Interface) []api.InterfaceStatus {
+func (l *LibvirtDomainManager) InterfacesStatus() []api.InterfaceStatus {
```

Comment:
whole func equal simply to `l.agentData.GetInterfaceStatus()` isn't it ?

Reply:
Hmm, right. I should remove the condition.
Done